server/templates/browser/node.html

```diff
@@ -23,17 +23,13 @@
    
    {% block head %}
      <link rel="stylesheet" href={{url_for('static', filename='css/kg.min.css', t=config['GAE_VERSION'])}} >
```

Comment:
we should try removing kg.min.css at some point, and migrate the styles we want to keep... for a later PR

Reply:
yes, will do